Jenkinsfile

```diff
@@ -120,15 +123,15 @@ pipeline {
 					junit '**/*TEST*.xml'
 					step([$class: 'CoberturaPublisher', autoUpdateHealth: false, autoUpdateStability: false, coberturaReportFile: "$JAVA_SRC/coverage*.xml", failUnhealthy: false, failUnstable: false, maxNumberOfBuilds: 0, onlyStable: false, sourceEncoding: 'ASCII', zoomCoverageChart: false, failNoReports: false])
 
-					// stop/kill emulator
-					sh "./buildScripts/build_helper_stop_emulator"
+					sh './gradlew stopEmulator'
+					archiveArtifacts 'logcat.txt'
 				}
 			}
 		}
 
 		stage('Build Debug-APK') {
 			steps {
-				sh "./buildScripts/build_step_create_debug_apk"
+				sh './gradlew clean assembleAndroidTest'
```

Comment:
This might need to be changed to 'assembleDebug' after #540 is merged, (or in #540) if this gets merged first.

Reply:
#540 is merged, please change this to "assembleDebug"